### DISCUSSION TITLE
io_uring support needed

### DISCUSSION THREAD

**Discussion by osevan** [openzfs/zfs#13442] [Q&A] [answered]
https://www.phoronix.com/scan.php?page=news_item&px=UBD-User-Space-IO_uring-Block

What you think about. .

Could zfs be faster with io-uring? 

I guess yes.

Thanks and

Best regards

**Comment by gmelikov** [comment, marked as answer]
Duplicate of https://github.com/openzfs/zfs/issues/8716

**Reply by osevan** [reply]
Thank you very much
